### DISCUSSION TITLE
mmWave Human Static Presence Module Lite | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#86] [Q&A] [answered]
# mmWave Human Static Presence Module Lite | Seeed Studio Wiki

mmWave Human Static Presence Module Lite

https://wiki.seeedstudio.com/Radar_MR24HPC1/

**Comment by jakabo27** [comment, marked as answer]
FYI for anyone trying to use the J-Link method, you will need a license for J-Flash to fully follow their Firmware Update tutorial using J-Link.  I borrowed a coworker's J-Link and tried to follow it but don't have a license.  I tried using J-Flash Lite and it should work, but it only erases the chip then fails on the programming device step.  So now I have a dud board because it erased the flash and failed to program it.  I used a lab power supply with a 2 amp limit and even upped it to 5.2V just in case a momentary voltage sag was causing the failure, but no luck.  Hopefully people that ordered more recent versions of the chip don't need to flash new firmware on the device.

**Reply by MatthewJeffson** [reply]
Thank you for the notice! It is very kind of you!
